### ISSUE TITLE
No Icons by Hourly forcast

### ISSUE THREAD

**Issue by Nulista** [Bodmer/DarkSkyWeather#11]
Hello,I have changed line 515 in darkSkyWeather.cpp

i add this lines to receive icons an is works:
    if (currentKey == "icon") hourly->icon[hourly_index] = iconIndex(val);
    else

Is that Oke?

mvg
Ludo

**Comment by Bodmer** [comment]
You can make any changes you wish!

You also need to [add the icon variable to the struct](https://github.com/Bodmer/DarkSkyWeather/blob/master/Data_Point_Set.h#L59). Then decide what you wish to do with that new information.

**Comment by Nulista** [comment]
Oke thx,

 

Yes I add the icon variable to the structure.

 

Thx a lot. You do a great job.

I read on the internet that Accuweather is more accurate than darksky….

What is your experience?

Accuweather has only 50 datarequest for free from the server.

Buts is oke for home use..

 

Ludo

 

Van: Bodmer [mailto:notifications@github.com] 
Verzonden: dinsdag 29 januari 2019 23:25
Aan: Bodmer/DarkSkyWeather <DarkSkyWeather@noreply.github.com>
CC: Nulista <ludo.nulens@pandora.be>; Author <author@noreply.github.com>
Onderwerp: Re: [Bodmer/DarkSkyWeather] No Icons by Hourly forcast (#11)

 

You can make any changes you wish!

You also need to add the icon variable to the struct <https://github.com/Bodmer/DarkSkyWeather/blob/master/Data_Point_Set.h#L59> . Then decide what you wish to do with that new information.

**Comment by Bodmer** [comment]
Hi,
I have not tried Accuweather although I did sign up for an account. The 50 per day restriction is rather low when you are trying to test code. I did intend to copy a response then setup a responder to send the same message for test purposes but never got around to trying that.

I am a bit short of time but now I have the DarkSkyWeather library running it should be quite quick to adapt. It is on my long "to do" list!
